src/test/java/edu/ksu/canvas/attendance/submitter/CanvasAssignmentAssistantUTest.java

```diff
@@ -174,9 +175,9 @@ public void deleteAssignmentInCanvasAssignmentNotFoundError() throws Exception {
 
         try {
             canvasAssignmentAssistant.deleteAssignmentInCanvas(COURSE_ID, oauthToken);
-        } catch (AttendanceAssignmentException exception) {
+        } catch (Exception exception) {
             LOG.warn("There was an error when deleting the Assignment. The following exception has been thrown: " + exception);
-            Assert.assertEquals(AttendanceAssignmentException.Error.DELETION_ERROR, exception.error);
+            Assert.assertEquals("java.lang.NullPointerException", exception.toString());
```

Comment:
Don't assert an exception type based on its `toString` value. Either change this line to use hamcrest matchers like so
```java
Assert.assertThat(exception, instanceOf(NullPointerException.class));
```
or if this test really does expect an exception, then you should generally change the `@Test` annotation on the method to 
```java
@Test(expected = NullPointerException.class)
```
But since this is throwing `NullPointerException` (the scourge of Java development) which can be thrown by far too many things unlike more specific Exceptions like what that line used to be, I'd rather you did this check inline at this specific point, so that some other NPE nearby doesn't cause a false positive.